modules/UpgradeWizard/language/en_us.lang.php

```diff
@@ -249,6 +249,7 @@
     'ERROR_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of SuiteCRM: ',
     'ERROR_PHP_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of PHP: ',
     'ERROR_SUITECRM_VERSION_INCOMPATIBLE' => 'The uploaded file is not compatible with this version of SuiteCRM: ',
+    'ERROR_FLAVOR_INCOMPATIBLE' => 'The uploaded file is not compatible with this flavor (Community Edition, Professional, or Enterprise) of SuiteCRM: ',
```

Comment:
We have flavors of SuiteCRM? We have paid versions? 😱

Reply:
Tbh I haven't noticed this. The upgrade script did that.
@mattlorimer ?

Reply:
That string was removed here: https://github.com/salesagility/SuiteCRM/pull/5482
from:
install/language/en_us.lang.php
modules/UpgradeWizard/language/en_us.lang.php

Reply:
It seems I've used an older upgrade pack for testing [although I was 7.10], but now it's fixed.
Thank you guys!